appinventor/components/src/com/google/appinventor/components/runtime/util/ErrorMessages.java

```diff
@@ -385,6 +389,14 @@
         "Cannot detect light level when the DetectColor property is set to True.");
     errorMessages.put(ERROR_NXT_INVALID_GENERATE_COLOR,
         "The GenerateColor property is limited to None, Red, Green, or Blue.");
+    errorMessages.put(ERROR_NXT_INVALID_MOTOR_MODE,
+        "%s is an invalid NXT motor mode.");
```

Comment:
Should be `%d` because the argument being provided is an `int`. This is true of the other messages added here as well.